### PR TITLE
extracts temperature conversion to separate class

### PR DESCRIPTION
**Pull request by tiagomartinho**
I extracted the conversion of temperature to a separate class in order to remove the comment and have a clear intent, what do you think?

### CODE REVIEW

File: SwiftWeather/TemperatureConverter.swift [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+class TemperatureConverter {
```
Comment (JakeLin) [inline SwiftWeather/TemperatureConverter.swift]: I think we can use `struct` here

**Comment by JakeLin** [comment]
@tiagomartinho I like it, can you use two spaces as one tab? Since we use that in the other places.

**Comment by tiagomartinho** [comment]
done! I also used a struct for TemperatureConverter as you suggested, even if i'm not sure if it makes a difference since it has only static methods

**Comment by JakeLin** [comment]
@tiagomartinho thanks for your PR, well done. As you said, I don't think they are different since they are `static`. To make thing simple for me, I always start with `struct`, can change later if needed.

**Pull request merged**